components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -3385,4 +3385,22 @@ private class ConfigElements {
         private static final String DEVICE_CODE_KEY_SET = "KeySet";
     }
 
+    /**
+     * Check whether introspecting tokens from different tenants is allowed.
+     *
+     * @return true/false Allow or block cross tenant token inspection.
+     */
+    public static boolean isCrossTenantTokenInspectionAllowed() {
+
+        String allowCrossTenantTokenInspectionConfigValue = IdentityUtil.getProperty(
```

Comment:
shall we get the configs similar to other configs in the class and follow the same pattern?

Reply:
can you check isTokenCleanupEnabled config

Reply:
resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1846/commits/b21657e856b2ce6c8b474995ac3a520e3f1a36f9

Reply:
resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1846/commits/b21657e856b2ce6c8b474995ac3a520e3f1a36f9